css/main.css

```diff
@@ -173,6 +172,7 @@ body {
     margin-right: auto;
     background: #F1F1F1;
     border-radius: 5px;
+    padding: 50px 0 50px 0;
```

Comment:
what do these changes do? won't this break the center panel?

Reply:
This adds padding to the top and bottom of the simple banner. In this case, we don't need to use responsive units because it looks fine on both desktop and mobile using the same padding. The point of adding this is to ensure that the text does not touch the top and bottom of the banner to make it more readable.

Without padding: 
![127 0 0 1_5500_ (1)](https://user-images.githubusercontent.com/16531060/98896632-70604e80-2477-11eb-84dc-5ea912c5a67c.png)

With padding (mobile):
![127 0 0 1_5500_](https://user-images.githubusercontent.com/16531060/98896631-6f2f2180-2477-11eb-8f63-98242c49f634.png)

With padding (desktop):
![127 0 0 1_5500_(FHD)](https://user-images.githubusercontent.com/16531060/98896710-a271b080-2477-11eb-8ea2-c199054eba84.png)